### PR TITLE
feat(loss)/add different operator types for cross_entropy

### PR DESCRIPTION
**Pull request by yingtongxiong**
## Motivation
1. support different operator types for cross_entropy
image
2. add op_type config
`
loss = dict(
    label_smoothing=0,
    op_type="flash_vocab_parallel",
)
`

reference this [https://github.com/pjlab-sys4nlp/InternTrain/pull/148](https://github.com/pjlab-sys4nlp/InternTrain/pull/148)

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.

### CODE REVIEW

**Comment by sunpengsdu** [comment]
@yingtongxiong merge下develop分支过CI吧

**Comment by yingtongxiong** [comment]
> @yingtongxiong merge下develop分支过CI吧

好的

**Comment by yingtongxiong** [comment]
> @yingtongxiong merge下develop分支过CI吧

@sunpengsdu 已经好了

**Review by sunpengsdu — APPROVED** [review]
